root/frontend/src/constants/CardColumn.ts

```diff
@@ -7,4 +7,6 @@ const sizeMap = new Map<number, number>([
   [6, 250],
 ]);
 
-export default sizeMap;
+const screenHeight: number = window.innerHeight - 130;
```

Comment:
If we're set on `130` being the height of the header, then we should probably have this value in a constant file which header should also reference.

As an aside, is this the only way to get this height? I'm not particular on pulling this sort of data that should be passed down through components of the DOM (this seems like pulling data from the top of the DOM). Something to consider, I'll leave that decision to you though.